app/modules/intelligence/agents/base_agent_service.py

```
@@ -0,0 +1,22 @@
+import os
+from typing import List
+
+from app.modules.intelligence.agents.agents_schema import AgentInfo
```

Ruff (0.8.2)

`app.modules.intelligence.agents.agents_schema.AgentInfo` imported but unused

Remove unused import: `app.modules.intelligence.agents.agents_schema.AgentInfo`

(F401)